src/@next/CurrencyInput/CurrencyInput.tsx

```diff
@@ -47,6 +55,36 @@ export const CurrencyInput = React.forwardRef<
     return Number.isNaN(cleanedValue) ? 0 : cleanedValue;
   };
 
+  const getCurrencySymbol = (
+    locale: string,
+    currencyCode: string,
+    currencySymbol?: string
+  ) => {
+    if (currencySymbol) {
+      return currencySymbol;
+    } else if (currencyCode) {
```

Comment:
we don't need this else if section since we have invert the if

Reply:
Updated